geode-wan/src/main/java/org/apache/geode/internal/WANDistributedSystemService.java

```diff
@@ -20,6 +20,7 @@
 
 import org.apache.geode.distributed.internal.DistributedSystemService;
 import org.apache.geode.distributed.internal.InternalDistributedSystem;
+import org.apache.geode.internal.classloader.ClassPathLoader;
```

Comment:
Is this addition for build purpose...Not seeing any changes in the class file.

Reply:
This is because the ClassPathLoader was moved
https://github.com/apache/geode/pull/6144/files#diff-4bd2f197c6e3442a0906cea9b03ecd0b0bdc5d74070d2e59c2e6c45e21f5ec47